Framework/DataHandling/src/LoadSampleShape.cpp

```diff
@@ -0,0 +1,201 @@
+#include "MantidDataHandling/LoadSampleShape.h"
+
+#include "MantidAPI/AnalysisDataService.h"
```

Comment:
You aren't using this either, just some other include that are getting included by accident